src/Microsoft.AspNetCore.Blazor/Json/SimpleJson/SimpleJson.cs

```diff
@@ -610,6 +618,13 @@ public static string SerializeObject(object json, IJsonSerializerStrategy jsonSe
 
         public static string SerializeObject(object json)
         {
+            CurrentJsonSerializerStrategy.SetPropertyNamingScheme(DefaultPropertyNamingScheme);
+            return SerializeObject(json, CurrentJsonSerializerStrategy);
+        }
+
+        public static string SerializeObject(object json, int propertyNaming)
```

Comment:
Can we make the `propertyNaming` type here `PropertyNaming` instead of `int`? Otherwise it's hard to make sense of what's going on.

You could move the `PropertyNaming` enum into the `SimpleJson.cs` file so it can be in the same namespace.

Reply:
Not a problem. I guess I was being too stubborn on trying to follow how SimpleJson was coded xD. Will move the enum here so that we can avoid unnecessary duplication of code as per feedback.